backend/src/db/fixture.js

```diff
@@ -0,0 +1,26 @@
+import User from "./entities/User";
+import Post from "./entities/Post";
+
+const peter = new User({
+  name: "Peter",
+  email: "peter@widerstand-der-pinguine.ev",
+  id: "b5bc19f4-164e-49c3-8045-7105580b43be",
+});
+
+const petersPassword = "peter1";
+
+const brother = new User({
+  name: "Peter's Bruder",
+  email: "augustus@coalition-der-waschbaere.ev",
+  id: "e8d085c6-9d3b-4762-8f71-12dc28bb45c7",
+});
+
+const brothersPassword = "brother1";
+
+const petersPost = new Post({
+  title: "Pinguine sind keine Vögel",
+  author: peter,
+  id: "613e8596-b432-4b74-89d4-06709644d8ee",
+});
+
+export default { peter, brother, petersPassword, brothersPassword, petersPost };
```

Comment:
although i like the approach of refactoring the dummy db, i think its better to put them into the test files directly. I guess the best approach is to use the factory Pattern and just create FactoryObjects here, but im not sure if graphQL gives the opportunity to do this